pocs/utils/library.py

```diff
@@ -14,6 +14,10 @@ def load_library(name, path=None, logger=None):
     Args:
         name (str): name of the library (without 'lib' prefix or any suffixes, e.g. 'fli').
         path (str, optional): path to the library e.g. '/usr/local/lib/libfli.so'.
+        mode (int, optional): mode in which to load the library, see dlopen(3) man page for
+            details. Should be one of ctypes.RTLD_GLOBAL, ctypes.RTLD_LOCAL, or
+            ctypes.DEFAULT_MODE. Default is ctypes.DEFAULT_MODE.
+        logger (logging.Logger, optional): logger to use.
```

Comment:
What happens if `mode=None`?

Reply:
If you explicitly set it to `None` you get something like
```
In [7]: libc = ctypes.CDLL("libc.so.6", mode=None)                                                                                     
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-9f402658a1ee> in <module>
----> 1 libc = ctypes.CDLL("libc.so.6", mode=None)

~/miniconda3/envs/huntsman/lib/python3.8/ctypes/__init__.py in __init__(self, name, mode, handle, use_errno, use_last_error, winmode)
    371 
    372         if handle is None:
--> 373             self._handle = _dlopen(self._name, mode)
    374         else:
    375             self._handle = handle

TypeError: an integer is required (got type NoneType)
```

If you simply don't set it it defaults to `ctypes.DEFAULT_MODE`, which is equivalent to `RTLD_LOCAL` in Linux and `RTLD_GLOBAL` in OS X.

Reply:
Hmmm. I should probably fix this. `pocs.utils.library.load_library("c")` works just fine, loading `libc` with the default mode, but `pocs.utils.library.load_library("c", mode=None)` fails. I think I should just have this function replace a `mode` of `None` with `ctypes.DEFAULT_MODE`.